packages/calcite-components/src/components/input-number/input-number.e2e.ts

```diff
@@ -1490,6 +1541,57 @@ describe("calcite-input-number", () => {
     expect(await input.getProperty("value")).toBe(numberStringFormatter.localize(initialValue));
   });
 
+  it("allows editing numbers that start with zeros and group separator", async () => {
+    const page = await newE2EPage();
+    await page.setContent(html`<calcite-input-number></calcite-input-number>`);
```

Comment:
This test says it is using a group separator, but the attribute is missing from the component

```suggestion
    await page.setContent(html`<calcite-input-number group-separator></calcite-input-number>`);
```